src/Microsoft.DotNet.Compiler.Common/LibraryExporterExtensions.cs

```diff
@@ -1,16 +1,23 @@
-﻿using System.Collections.Generic;
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
 using System.IO;
-using System;
 using System.Linq;
-using Microsoft.DotNet.ProjectModel;
 using Microsoft.DotNet.ProjectModel.Compilation;
 using Microsoft.DotNet.ProjectModel.Graph;
-using Microsoft.Extensions.DependencyModel;
 
 namespace Microsoft.DotNet.Cli.Compiler.Common
 {
     public static class LibraryExporterExtensions
     {
+        public static IEnumerable<LibraryExport> GetProjectDependencies(this LibraryExporter exporter)
```

Comment:
I am not a big fan of this name. It is confusing to me if we are getting all the dependencies that the project depends on or all the dependencies of type project.

Maybe GetAllProjectTypeDependencies?